app/javascript/modules/listingDetailsAside/ListingDetailsProcess.tsx

```diff
@@ -10,28 +10,59 @@ export interface ListingDetailsProcessProps {
 
 export const ListingDetailsProcess = ({ listing }: ListingDetailsProcessProps) => {
   return (
-    !!listing.Lottery_Date &&
-    dayjs(listing.Lottery_Date) > dayjs() &&
-    !listing.LotteryResultsURL && (
-      <EventSection
-        events={[
-          {
-            dateString: localizedFormat(listing.Lottery_Date, "LL"),
-            timeString: dayjs(listing.Lottery_Date).format("hh:mma"),
-            note: getEventNote({
-              City: listing.Lottery_City,
-              Street_Address: listing.Lottery_Street_Address,
-              Venue: listing.Lottery_Venue,
-            }),
-          },
-        ]}
-        headerText={"Public Lottery"}
-        sectionHeader={true}
+    <>
+      {!!listing.Lottery_Date &&
+        dayjs(listing.Lottery_Date) > dayjs() &&
+        !listing.LotteryResultsURL && (
+          <EventSection
+            events={[
+              {
+                dateString: localizedFormat(listing.Lottery_Date, "LL"),
+                timeString: dayjs(listing.Lottery_Date).format("hh:mma"),
+                note: getEventNote({
+                  City: listing.Lottery_City,
+                  Street_Address: listing.Lottery_Street_Address,
+                  Venue: listing.Lottery_Venue,
+                }),
+              },
+            ]}
+            headerText={"Public Lottery"}
+            sectionHeader={true}
+          />
+        )}
+      {/* <GetApplication /> */}
+      {/* <SubmitApplication /> */}
+      <Contact
+        sectionTitle={t("contactAgent.contact")}
+        contactAddress={{
+          street: listing.Leasing_Agent_Street,
+          city: listing.Leasing_Agent_City,
+          state: listing.Leasing_Agent_State,
+          zipCode: listing.Leasing_Agent_Zip,
+        }}
+        additionalInformation={
+          listing.Office_Hours
+            ? [
+                {
+                  title: t("contactAgent.officeHours"),
+                  content: listing.Office_Hours,
+                },
+              ]
+            : undefined
+        }
+        contactEmail={listing.Leasing_Agent_Email}
+        contactName={listing.Leasing_Agent_Name}
+        contactPhoneNumber={`Call ${listing.Leasing_Agent_Phone}`}
```

Comment:
We actually don't have translations for this currently and in the angular is it hard coded as `Call` - but then does end up getting translated. @christine-sfg Do you have a preference for how we handle this? (1) Leave it hardcoded here as well and see if we can google translate it as it was or (2) Add just an english string for now